tracer/test/test-applications/integrations/Samples.InstrumentedTests/Vulnerabilities/InstrumentationTestsBase.cs

```diff
@@ -0,0 +1,140 @@
+// <copyright file="InstrumentationTestsBase.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using FluentAssertions;
+
+namespace Samples.InstrumentedTests.Iast.Vulnerabilities;
```

Comment:
seems to have been mainly copied / pasted from https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/test-applications/Samples.Shared/SampleHelpers.cs

I'm wondering if there's some way we could share / reuse that code instead.

Reply:
>I'm wondering if there's some way we could share / reuse that code instead

The sample helpers are a "shared" project, which means they're available in any project that references Samples.Shared (which is implicitly _all_ test applications in this folder by default).

The code here appears to use `SampleHelpers` already, so seems like a legitimate decision to keep the IAST-specific reflection here. That seems reasonable to me for now, and should be easy to move it to `SampleHelpers` later if the need arises 🙂